mleap-spark/src/main/scala/org/apache/spark/ml/bundle/ops/feature/BucketedRandomProjectionLSHOp.scala

```diff
@@ -33,17 +33,14 @@ class BucketedRandomProjectionLSHOp extends SimpleSparkOp[BucketedRandomProjecti
     override def load(model: Model)
                      (implicit context: BundleContext[SparkBundleContext]): BucketedRandomProjectionLSHModel = {
       val ruv = model.value("random_unit_vectors").getTensorList[Double].map(_.toArray).map(Vectors.dense)
-      val m = new BucketedRandomProjectionLSHModel(uid = "",
-        randUnitVectors = ruv.toArray)
+      val m = new BucketedRandomProjectionLSHModel(uid = "", randUnitVectors = ruv.toArray)
       m.set(m.bucketLength, model.value("bucket_length").getDouble)
-
-      m
```

Comment:
I think this needs to be here, no?

Reply:
no need, the result of `m.set` is `m`. It also changes `m` itself, so both ways are fine

Reply:
👍